packages/compiler-sfc/__tests__/compileStyle.spec.ts

```diff
@@ -161,6 +161,43 @@ describe('SFC scoped CSS', () => {
     `)
   })
 
+  // #10511
+  test(':is and :where with same pre-selector should have right priority', () => {
+    expect(
+      compileScoped(`.div { color: red }; .div:where(:hover) { color: blue }`),
```

Comment:
should be `{ color: red; }` not `{ color: red };`

Reply:
That's indeed, fixed it